### PR TITLE
Add a Loading button

### PR DESCRIPTION
**Pull request by akshatgarg12**
Issue:

### CODE REVIEW

File: BUTTON_IDEAS.md [hunk 1]
```diff
@@ -60,6 +60,7 @@ When adding ideas, please add them in the right category (or add a new one if yo
 | &#9744; | **Dark Mode Toggle**  | A button toggling dark mode.                               | [Harshit Virmani](https://github.com/hvirmani)    |
 | &#9745; | ~~**Disabled Buttons**~~  | A button that has disabled style and non-clickable ability | [Zahra Abdullah](https://github.com/zahra-abdullah) |
 | &#9744; | **Spooky Button** | A button with dramatic/spooky touch to it for special website designs. | [Shristi Singh](https://github.com/confusedcoder1)    |
+| &#9744; | **Wheel button** | A wheel as a button, rotating on hover | [Akshat Garg](https://github.com/akshatgarg12)    |
```
Comment (shahednasser) [inline BUTTON_IDEAS.md:63]: Rename it to Loading Button and change description to a button with CSS loading

Comment (akshatgarg12) [reply BUTTON_IDEAS.md:63]: okay.
but I believe this wheel button is also possible in CSS?

Comment (shahednasser) [reply BUTTON_IDEAS.md]: It is but I think it requires some nested element and currently, we're not going for that. We want the button as simple as possible.

**Comment by shahednasser** [comment]
Great thank you!

@all-contributors please add @akshatgarg12 for idea

File: BUTTON_IDEAS.md (hunk 1 above)
Comment (akshatgarg12) [reply BUTTON_IDEAS.md]: okay thanks!

**Pull request merged**